### DISCUSSION TITLE
Checkmark for viewed shows - Reposting

### DISCUSSION THREAD

**Discussion by mirtim1** [umbrellaplug/umbrellaplug.github.io#280] [Q&A]
Never saw any response to this question so I am reposting: Does the check mark for viewed shows feature work for anyone? For me it is somewhat random. Usually the first time I watch a show in a series, I get a check mark and then never again. Just talking about in the base Umbrella interface. I do not use Trakt or IMDB. IO get the same effect on Fire devices and on my Macs.  Is this unique to my setup, a known limitation or user error?

**Comment by umbrellaplug** [comment]
I don't really pay close enough attention to this. I will need to test and see how the check marks watch. I'm assuming this is in estuary stock skin. What version of kodi are you running?

**Comment by mirtim1** [comment]
No, Actually I am running confluence. Maybe thats the problem. It is Kodi 20.0.

**Reply by mirtim1** [reply]
OK, I switched to Estuary and the checkmarks are still inconsistent. It's a minor thing but it would be nice to have this working.
